vignettes/montagu_user_guide.Rmd

```diff
@@ -494,11 +675,11 @@ params
 
 ```
 ##   run_id param_1 param_2
```

Comment:
are these generated randomly? if so `set.seed` will help cut down on noise

Reply:
I do `params <- data.frame(run_id = 1:5, param_1 = sample(5), param_2 = sample(5))` 
at the moment...

Reply:
Have prefixed that block with
```{r include = FALSE}
set.seed(12345)
```